admin/javascript/src/sections/campaign-admin/controller.js

```diff
@@ -52,13 +53,21 @@ class CampaignAdminContainer extends SilverStripeComponent {
     return (
       <div className="cms-middle no-preview">
         <div className="cms-campaigns collapse in" aria-expanded="true">
-          <NorthHeader />
-          <FormAction
-            label={i18n._t('Campaigns.ADDCAMPAIGN')}
-            icon={'plus-circled'}
-            handleClick={this.addCampaign}
-          />
-          <FormBuilder schemaUrl={schemaUrl} createFn={this.createFn} />
+          <NorthHeader>
+            <h2 className="text-truncate north-header__heading">Campaigns</h2>
```

Comment:
Come to think of it, this should probably be passed down from the server-side config - since that's the way we configure ModelAdmins for example. Let's hold off on this, hardcoding is OK for now.